Documentation/ReleaseNotes.md

```diff
@@ -144,6 +148,14 @@ Specify the desired raycast provider in the Input System's configuration profile
 
 ![Selecting the Raycast provider](Images/ReleaseNotes/SelectRaycastProvider.png)
 
+**Improved Solvers**
```

Comment:
```suggestion
**Improved solvers**
```

I don't think we should capitalize solvers - https://microsoft.github.io/MixedRealityToolkit-Unity/version/prerelease/2.0.0_stabilization/Documentation/Contributing/DocumentationGuide.html#capitalization